tests/src/Interop/ClassicCOM/ClassicCOMUnitTest.csproj

```diff
@@ -1,47 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please don't delete these sceanrio unless you add them in another place. These testcase are testing reflection load interop assembly in x-platform..

Reply:
I added back the scenarios that are not being covered in dotnet/corefx#30928.